packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -38,17 +38,17 @@ const propTypes = {
     GranularityOptions.DATE,
   ]),
 
-  /**
-   * Name of the granularity select. The name should be unique.
-   */
-  granularitySelectName: PropTypes.string.isRequired,
-
   /**
    * A callback function to execute when a granularity is selected.
    * The first parameter is the changed granularity value.
    */
   granularitySelectOnChange: PropTypes.func,
 
+  /**
+   * The id of the onset picker. Used as the base for other required id/name in sub-components.
+   */
+  id: PropTypes.string.isRequired,
```

Comment:
Typically the addition of a new required prop would be considered non-passive. Do we want to bump the version, or is the consumption base so low that we're comfortable not doing so?

Reply:
If a new required prop is added this will have to be major versioned.

Reply:
Ah yes the plan was to go to the next major version, I'll update that

Reply:
Updated to 3.0.0

Reply:
You don't need to bump the version yourself, that'll happen during the actual release. You just need to add a `### Breaking Change` header to your changes in the CHANGELOG so that the person doing the release knows to bump the major version.

Reply:
Ah alright, I've reverted it to 2.15.0 and included a breaking change header to the changelog.